app/models/middleware_server.rb

```diff
@@ -1,5 +1,6 @@
 class MiddlewareServer < ApplicationRecord
   belongs_to :ext_management_system, :foreign_key => "ems_id"
+  belongs_to :middleware_server_group, :foreign_key => "server_group_id"
```

Comment:
The standalone servers have the `server_group_id` set to `nil` so I guess, it works.